src/oatpp/core/data/mapping/type/Object.hpp

```diff
@@ -102,6 +102,10 @@ class BaseObject : public oatpp::base::Countable {
        */
       std::string pattern = "";
 
+      /**
+       * Format.
+       */
+      std::string format = OATPP_FLOAT_STRING_FORMAT;
```

Comment:
Let's initialize it with an empty string

```cpp
std::string format = "";
```

Then, when serializing we'll check if `format` isn't empty and choose the proper format.